src/main/webapp/test.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Is this file still required?

Reply:
I removed in the next PR; for this PR, yes, it was required.

Reply:
Why was it required for this PR? I also see that this is the only place that uses the script.js.

Reply:
This PR is the most basic version of comments, which I used to test if the get/fetch servlets flow actually worked - a user will input comments on test.html, which is just a simple, non-styled form. Only in the next PR did I add the functionality to recipe.html, where it's supposed to be.

Now that you mention it, though, perhaps this PR is a bit unnecessary, since I just delete test.html later on. Should I close it?

Reply:
I think you should delete the test and script in this PR itself, since everything else is reviewed and seems okay.

Reply:
Oh, I found your comment! Weird that it only shows up in file view. I deleted the test and script according to your suggestion; will re-submit PR.